modules/op-release-notes-1-12.adoc

```diff
@@ -0,0 +1,192 @@
+
+* With this update, on the GitHub status check page, the table that shows the status of all tasks includes the display name of every task.
+
+* With this update, you can configure the `tags push` event in a pipeline run on GitLab.
+
+* Release Notes: With this update, you can use the `target_url` and `source_url` fields in {pac} Common Expression Language (CEL) expression filtering annotations to filter the request for specific target or source.
+
+* With this update, when configuring fetching a remote GitHub URL using a token, you can include a branch name that contains a slash. You must encode the slash within the branch name as `%2F` to ensure proper parsing by {pac}m as in the following example URL: `\https://github.com/organization/repository/blob/feature%2Fmainbranch/path/file`. In this example URL, the branch name is `feature/mainbranch` and the name of the file to fetch is `/path/file`.
+
+* In this release, you can use `--v1beta1` flag in the `tkn pac resolve` command. Use this flag if the pipeline run is generated with the `v1beta1` apiVersion schema.
+
+[id="breaking-changes-1-12_{context}"]
+== Breaking changes
+
+* With this update, you cannot use the `openshift-operators` namespace as the target namespace for installing {pipelines-shortname}. If you used the `openshift-operators` namespace as the target namespace, change the target namespace before upgrading to OpenShit Pipelines Operator version 1.12. Otherwise, after the upgrade you will not be able to change any configuration settings in the `TektonConfig` CR except the `targetNamespace` setting.
+
+* With this update, the new `spec.pipeline.performance.replicas` parameter controls the number of replicas that is created for every pod for a pipeline run or task run. If you previously set the replica counts in your deployment manually, you must use this setting to control the replica counts after this update
+
+* With this update, the following parameters in the `TektonResult` CR are no longer supported:
+** `db_user`
+** `db_password`
+** `s3_bucket_name`
+** `s3_endpoint`
+** `s3_hostname_immutable`
+** `s3_region`
+** `s3_access_key_id`
+** `s3_secret_access_key`
+** `s3_multi_part_size`
++
+You must provide these parameters using secrets. You must delete and re-create the `TektonResult` CR after upgrading to {pipelines-shortname} 1.12 to provide these parameters.
+
+* With this update, the `tkn pac bootstrap` command supports the `--github-hostname` flag. The `--github-api-url` flag is deprecated.
+
+[id="known-issues-1-12_{context}"]
+== Known issues
+
+* If limits ranges are configured for a namespace, but pod ephemeral storage is not configured in the limit ranges, pods can go into an error stage with the message `Pod ephemeral local storage usage exceeds the total limit of containers 0`.
```

Comment:
It's called `limit range` not `limits range` https://kubernetes.io/docs/concepts/policy/limit-range/